song-server/src/main/resources/db/migration/V1_9__filetype_enum_add_txt.sql

```diff
@@ -0,0 +1,20 @@
+ALTER TYPE file_type RENAME TO _file_type;
```

Comment:
I'm not very familiar with enums, but I thought it would just be something like this: 
`ALTER TYPE enum_name ADD VALUE 'test4' AFTER 'test3';`

Reply:
yeah I looked up the same and had prepared that, but in migration v1_4 it was done this way so I thought to stay consistent with what worked previously...